src/components/Editor/index.js

```diff
@@ -156,6 +156,20 @@ class Editor extends Component {
     codeMirrorWrapper.addEventListener("mouseup", e => this.onMouseUp(e, ctx));
     codeMirrorWrapper.addEventListener("mouseover", e => this.onMouseOver(e));
 
+    const toggleFoldMarkerVisibility = e => {
+      if (node instanceof HTMLElement) {
+        node
+          .querySelectorAll(".CodeMirror-guttermarker-subtle")
+          .forEach(elem => {
+            elem.classList.toggle("visible");
+          });
+      }
+    };
```

Comment:
We need `node` to get all fold markers and toggle class. I'm not sure if there is a better method here other than using `querySelectorAll`.